pkg/cache/cache.go

```diff
@@ -12,13 +12,18 @@ import (
 )
 
 const (
```

Comment:
maybe do some pseudo enum like in go https://stackoverflow.com/questions/71934060/go-how-to-use-enum-as-a-type